test/libsolidity/semanticTests/libraries/library_address_homestead.sol

```diff
@@ -0,0 +1,25 @@
+library L {
+    function f(uint256 a, uint256 b) external {
+        assert(a * a == b);
+    }
+}
+contract C {
+    function addr() public view returns (bool) {
+        return address(L) == address(0);
+    }
+    uint256[] s;
```

Comment:
What's this for?

Reply:
Sorry, leftover from another (non-working) version of this - I'll remove it.

Reply:
Removed.

Reply:
No worries, thought you were going to use it via `delegatecall` or similar

Reply:
I tried to pass this down to the library and have the library modify it - but I'm not sure we in fact add selectors for functions with storage arguments, so it's probably not possible, so I changed it to this weird assertion check only :-).